src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js

```diff
@@ -0,0 +1,122 @@
+import React, { PureComponent } from 'react';
+import {
+  PieChart, Pie, Sector, Cell, ResponsiveContainer,
+} from 'recharts';
+
+let COLORS_EVEN = [
```

Comment:
Can you rename it to DEFAULT_COLORS_EVEN, DEFAULT_COLORS_ODD for more code readability